Examples/Algorithms/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_subdirectory_if(DD4hepG4 ACTS_BUILD_EXAMPLES_DD4HEP AND ACTS_BUILD_EXAMPLES_GEANT4)
```

Comment:
Yes, because DD4hep and HepMC3 are not generally available in the CI builds. DD4hep is missing from the Ubuntu 20.04 image but will be added again (#129). HepMC3 is not available in LCG95 and it can only be dropped from the `EVERYTHING` flag once we drop the LCG95 support.